src/cli/cli_tcp.hpp

```diff
@@ -108,13 +121,19 @@ class TcpExample : private Output
                                                        otTcpEndpoint   **aAcceptInto);
     void HandleTcpAcceptDone(otTcpListener *aListener, otTcpEndpoint *aEndpoint, const otSockAddr *aPeer);
 
+#if OPENTHREAD_CONFIG_TLS_ENABLE
+    static void MbedTlsDebugOutput(void *ctx, int level, const char *file, int line, const char *str);
+#endif
+
     otTcpEndpoint mEndpoint;
     otTcpListener mListener;
 
     bool mInitialized;
     bool mEndpointConnected;
     bool mSendBusy;
     bool mUseCircularSendBuffer;
+    bool mUseTLS;
+    bool mTLSHandshakeComplete;
```

Comment:
nit: prefer camel case acronyms
```suggestion
    bool mUseTls;
    bool mTlsHandshakeComplete;
```

Reply:
I fixed this in the latest commit.